setup.py

```diff
@@ -49,7 +49,8 @@
         'Django>=2.2,<3.2',             # should be the same as wagtail
         'geocoder==1.38.*',
         'icalendar==4.0.*',
-        'wagtail==2.11.*',
+        'unidecode==1.*',
```

Comment:
Is there a reason for this new dependency?

Reply:
When I ran pytest without it, it said ModuleNotFound and broke it. unidecode is used in the blocks.py file. Is there another way to bring it in so that it's recognized as a module?

Reply:
Ok, I'm guessing wagtail may have had it as a dependency and no longer uses it in 2.12. I would say first let's see where/why we are using it... maybe it can be removed. See if wagtail 2.12 release notes mentions it at all.

Reply:
Found a note in 2.11 actually: "Remove unidecode and use anyascii in for Unicode to ASCII conversion" You are currently using it in wagtail_flexible_forms > blocks.py and it's being used in get_slug.

Reply:
Please make the corresponding replacement to anyascii in our code, if possible, to remain closer to wagtail and eliminate the extra dependency.

Reply:
done.